src/main/java/address/browser/HyperBrowser.java

```diff
@@ -74,7 +76,8 @@ private void initialiseHyperBrowser(){
         for (int i = 0; i < noOfPages; i++){
             EmbeddedBrowser browser;
             if (browserType == FULL_FEATURE_BROWSER){
-                browser = new JxBrowserAdapter(new Browser());
+                BrowserContext context = new BrowserContext(new BrowserContextParams("tmpTab" + i));
```

Comment:
What's the context for? The documentation states for data directory such as cookies. If that's the case, we shouldn't separate them as the same user will be using the different tabs and it's easier if the different tabs use the same cookies.